LICENSE

```diff
@@ -1,14 +1,20 @@
-The BSD Zero Clause License (0BSD)
+The MIT License (MIT)
 
-Copyright (c) 2020 Gatsby Inc.
 
```

Comment:
The MIT License says you need to have a line with Copyright and the year https://opensource.org/licenses/MIT

So Something like "Copyright 2020 The Geeks BlaBla team" as an example without ""